pycloudlib/azure/cloud.py

```diff
@@ -452,21 +459,27 @@ def delete_image(self, image_id):
                 "Error deleting %s. Request returned %d", image_id, resp_code
             )
 
+    def _get_image(self, release, image_map):
+        release = image_map.get(release)
+        if release is None:
+            msg = "No Ubuntu image found for {}. Expected one of: {}"
+            raise ValueError(
+                msg.format(release, " ".join(UBUNTU_DAILY_IMAGES.keys()))
+            )
+
+        return release
```

Comment:
Per above suggestions
```suggestion
        return image_id
```